client/signup/controller.js

```diff
@@ -245,6 +245,20 @@ export default {
 				experiment.variationName === 'treatment_scrambled';
 		}
 
+		// See: 1113-gh-Automattic/experimentation-platform for details.
+		const isGuidedFlow = flowName === 'guided';
+		if ( isGuidedFlow && ( ! userLoggedIn || ( userLoggedIn && isNewUser ) ) ) {
```

Comment:
> ! userLoggedIn

Looks like we're assigning users when they're logged out. Can we make assignment occur after the user has logged-in? See issue 7021 in `Automattic/dotcom-forge`